src/main/java/seedu/sudowudo/model/item/Item.java

```diff
@@ -57,200 +53,197 @@ public Item(Description desc, UniqueTagList tags) {
+     * @return boolean, whether the item is or isn't
+     * @@author A0092390E
+     */
+    @Override
+    public boolean is(String query) {
+        switch (query.toLowerCase()) {
+        case "done":
+            return this.getIsDone();
+        case "undone":
+            return !this.getIsDone();
+        case "event":
+            return this.getStartDate() != null;
+        case "task":
+            return this.getStartDate() == null;
+        case "overdue":
+            return this.is("task") && this.getEndDate() != null && this.getEndDate().isBefore(LocalDateTime.now());
+        case "item":
+            return true;
+        default:
+            return false;
+        }
     }
 
-    public Period getRecurInterval() {
-        return recurInterval;
-
-    }
-    public LocalDateTime getRecurEndDate() {
-        return recurEndDate;
-    }
-    
-	/**
-	 * Flexible property querying, to support listing and filtering
-	 * 
-	 * @return boolean, whether the item is or isn't
-	 * @@author A0092390E
-	 */
+    /**
+     * Returns the type of the item. Useful for Item cards and messages.
+     * 
+     * @return The type of the item, computed based on start/end dates.
+     *         [Event|Floating Task|Task]
+     * @@author A0092390E
+     */
     @Override
-	public boolean is(String query){
-    	query = query.toLowerCase();
-		switch (query) {
-		case "done":
-			return this.getIsDone();
-		case "undone":
-		    return !this.getIsDone();
-		case "event":
-			return this.getStartDate() != null;
-		case "task":
-			return this.getStartDate() == null;
-		case "overdue":
-			return this.is("task") && this.getEndDate() != null 
-			        && this.getEndDate().isBefore(LocalDateTime.now());
-		case "item":
-		    return true;
-		default:
-			return false;
-		}
+    public String getType() {
+        if (this.getStartDate() != null) {
+            return "Event";
+        } else if (this.getEndDate() == null) {
+            return "Floating Task";
+        } else {
+            return "Task";
+        }
     }
 
-	/**
-	 * Returns the type of the item. Useful for Item cards and messages.
-	 * 
-	 * @return The type of the item, computed based on start/end dates.
-	 *         [Event|Floating Task|Task]
-	 * @@author A0092390E
-	 */
-	@Override
-	public String getType() {
-		if (this.getStartDate() != null) {
-			return "Event";
-		} else if (this.getEndDate() == null) {
-			return "Floating Task";
-		} else {
-			return "Task";
-		}
-	}
-
-	/**
-	 * Replaces this Item's tags with the tags in the argument tag list.
-	 */
+    /**
+     * Replaces this Item's tags with the tags in the argument tag list.
+     */
     public void setTags(UniqueTagList replacement) {
         tags.setTags(replacement);
-		setChanged();
-		notifyObservers();
+        setChanged();
+        notifyObservers();
     }
 
-    
     @Override
-	public void setIsDone(boolean doneness) {
+    public void setIsDone(boolean doneness) {
         this.isDone = doneness;
```

Comment:
Change parameter `doneness` to `isDone` (it's my fault originally I'm sorry 😭 ) for coding standard